week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -13,45 +13,85 @@ sort method 😎)
 You don't have to worry about making this algorithm work fast! The idea is to get you to
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
+function filterArrays(arr) {

```

Comment:
One thing I wanted to highlight and forgot to mention on the first pass. Great job on extracting this into a separate function! Things like this make the code easier to read and maintain and will be crucial later down the line! Keep it up! 🥇

Reply:
thank you